apis/v1alpha1/shared_types.go

```diff
@@ -141,15 +127,17 @@ type NamespacedPeer struct {
 	// SameLabels is used to select a set of Namespaces that share the same values
 	// for a set of labels.
 	// To be selected a Namespace must have all of the labels defined in SameLabels,
-	// and they must all have the same value as the subject of this policy.
+	// AND they must all have the same value as the subject of this policy.
 	// If Samelabels is Empty then nothing is selected.
 	// +optional
 	// +kubebuilder:validation:MaxItems=100
 	SameLabels []string `json:"sameLabels,omitempty"`
 
 	// NotSameLabels is used to select a set of Namespaces that do not have a set
-	// of label(s). To be selected a Namespace must have none of the labels defined
-	// in NotSameLabels. If NotSameLabels is empty then nothing is selected.
+	// of label(s).
```

Comment:
suggestion: NotSameLabels is used to select a set of Namespaces that do not have certain values for a set of label(s).

Reply:
ack done.